### PR TITLE
Add on demand record count setting in preferences.#3275

### PR DESCRIPTION
**Pull request by yogeshmahajan-1903**

### CODE REVIEW

File: docs/en_US/preferences.rst [hunk 1]
```diff
@@ -440,6 +440,9 @@ preferences for copied data.
 * Specify the maximum width of the column in pixels when 'Columns sized by' is
   set to *Column data*. If 'Columns sized by' is set to *Column name* then this
   setting won't have any effect.
+* Specify the number of records to fetch in one batch in query tool when
+  query result set is large.Changing this value will over ride
```
Comment (adityatoshniwal) [inline docs/en_US/preferences.rst:444]: `over ride` should be `override`

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]
Other changes looks good to me.

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]
You probably missed the screenshot of prefs.

**Comment by yogeshmahajan-1903** [comment]
> You probably missed the screenshot of prefs.

It is already updated.

**Review by adityatoshniwal — APPROVED** [review]

**Pull request merged**